dist/non-amd/jquery.autotagging.js

```diff
@@ -200,7 +200,7 @@ jquery_autotagging_click_handler = function ($) {
       return window.open(href);
     };
     ClickHandler.prototype.elemClicked = function (e, options) {
-      var attr, attrs, domTarget, getClosestAttr, item, jQTarget, realName, subGroup, trackingData, value, _i, _len, _ref;
```

Comment:
I don't see a CoffeeScript change that removed this variable and the function assigned to it. Did that happen in a prior commit?